### ISSUE TITLE
Header in patient self-registration isn't covered by landmarks

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3983]
## Area or Component
Patient Experience - Self registration terms of service

## Quick Summary
"Register for your test"/facility name potentially not contained by landmarks

## Accessibility Violation
All page content should be contained by landmarks
https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome

## Action Requested
Add a landmark (`main`?) that contains the page headings.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/PatientHeader.tsx#L45-L45

## Original Deque Issue
region | Ensures all page content is contained by landmarks | "Register for your test"/facility name potentially not contained by landmarks | All page content should be contained by landmarks | https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome | moderate | No | No |   | [".padding-y-105"] | Fix any of the following:  Some page content is not contained by landmarks | <div class="bg-white padding-y-105"><div class="grid-container maxw-tablet"><h1 class="margin-top-0 margin-bottom-1">Register for your test</h1><h2 class="margin-y-0 text-normal">Via Elegante</h2></div></div> | cat.keyboard,best-practice | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T20:53:04.559Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by fzhao99** [comment]
I think this got taken care of by [4134](https://github.com/CDCgov/prime-simplereport/pull/4134)

Screenshot of prod with the relevant parts of the app highlighted in the DOM inspector
<img width="1052" alt="Screen Shot 2022-08-09 at 9 24 22 AM" src="https://user-images.githubusercontent.com/29645040/183674513-ebd755b9-cd2c-484c-ac77-49ec01a4fa72.png">

**Comment by emmastephenson** [comment]
Yep this should be good to go now, thanks Bob!
